tpkutils/__init__.py

```diff
@@ -341,15 +341,24 @@ def to_mbtiles(self, filename, zoom=None, tile_bounds=False, drop_empty=False):
             zoom = list(zoom)
             zoom.sort()
 
-            tiles = (
-                tile
-                for tile in self.read_tiles(zoom, flip_y=True)
-                if not (
-                    drop_empty and hashlib.sha1(tile.data).hexdigest() in EMPTY_TILES
-                )
-            )
-
-            mbtiles.write_tiles(tiles)
+            real_zooms = (
+                set()
+            )  # Zooms for which at least some tiles have not been dropped
```

Comment:
Minor nit: can you please move the comment to above `real_zooms` so there isnt' wrapping